gulpfile.js

```diff
@@ -131,92 +97,5 @@ gulp.task('publish', shell.task([
  *     'node_modules/@plentymarkets/terra-components' in target directory
  *
  **/
-gulp.task('npm-publish', function () {
-    increment = argv.increment ? argv.increment : 'patch';
-    preid = argv.preid ? argv.preid : '';
-
-    if(argv.level || argv.subversion)
-    {
-        console.log('-------------------------------------------------------------------');
-        console.log('----  Build not started. See gulpfile for further information. ----');
-        console.log('-------------------------------------------------------------------');
-
-        return;
-    }
-    else
-    {
-        return gulp.series(
-            changeVersion,
-            'clean-dist',
-            'compile-ts',
-            'copy-files',
-            'copy-fonts',
-            'copy-lang',
-            'copy-tslint-rules',
-            'publish'
-        );
-    }
-}());
-
-/**
- * define tasks for 'build-doc'
- */
-gulp.task('copy-to-terra-doc', function () {
-    return gulp.src(config.sources.dist)
-        .pipe(gulp.dest(config.destinations.terraComponentsDoc));
-});
-gulp.task('copy-api-to-terra-doc', function () {
-    return gulp.src('component-documentation/build/**/*.*')
-        .pipe(gulp.dest(config.destinations.terraComponentsDocBuild));
-});
-
-//copy components from dist to terra-component-doc
-gulp.task('copy-components-to-doc', function () {
-    return gulp.src('src/lib/components/**/**/example/*.ts')
-        .pipe(gulp.dest(config.destinations.terraComponentsDocComponents));
-});
-
-gulp.task('copy-markdown-to-doc', function () {
-    return gulp.src('src/lib/components/**/example/*.md')
-        .pipe(gulp.dest(config.destinations.terraComponentsDocComponents));
-});
-
-gulp.task('copy-icon-description-json', function () {
-    return gulp.src('src/lib/styles/iconDescription.json')
-        .pipe(gulp.dest(config.destinations.terraComponentsDocBuild));
-});
-gulp.task('copy-documentation-changelog', function () {
-    return gulp.src('component-documentation/documentation-changelog.json')
-        .pipe(gulp.dest(config.destinations.terraComponentsDocBuild));
-});
-gulp.task('dgeni', function () {
-    try {
-        var dgeni = new Dgeni([require('./component-documentation/index')]);
-        return dgeni.generate();
-    } catch (x) {
-        console.log(x.stack);
-        throw x;
-    }
-});
-
-gulp.task('generateJson', function (done)
-{
-    gulpTasks.buildJsonFile(paths.dataJsonOutputPath);
-    done();
-});
-
-/**
- * run "gulp build-doc" to let Dgeni generate api files and to create json data.
- */
-gulp.task('build-doc',
-    gulp.series(
-        'build',
-        'dgeni',
-        'generateJson',
-        'copy-to-terra-doc',
-        'copy-components-to-doc',
-        'copy-api-to-terra-doc',
-        'copy-markdown-to-doc',
-        'copy-icon-description-json',
-        'copy-documentation-changelog'
-    ));
+const release = series(changeVersion, copy, publish);
```

Comment:
Hier fehlt die Kompilierung der css Dateien
```suggestion
const release = series(changeVersion, compileCss, copy, publish);
```

Reply:
done